scss/_patterns_navigation.scss

```diff
@@ -244,7 +245,6 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
   .p-navigation__tagged-logo {
     display: flex; // to prevent logo link from expanding full width
     margin-right: 0;
-    min-width: 13rem;
```

Comment:
@ClementChaumel I didn't mean to remove it completely. We still need it on desktop. But on mobile we don't need max width I think.

So just wraping it in media query should work?